### ISSUE TITLE
consider moving itext handling to client

### ISSUE THREAD

**Issue by MartijnR** [SEL-Columbia/enketo#310]
Only after client XPath performance has been improved dramatically.

This makes sense for the following reasons:
- the current mechanism for translations in selects and itemsets is not very elegant. Moreover, often option lists are used multiple times and currently copied multiple which can cause the # of DOM elements to explode in some forms.
- XSLT transformation should be up to 85% quicker (to be checked with new sheet), which would have a noticeable impact on loading long forms for the first time and for loading previews (although the latter could be cached too, if it is likely previews are loaded multiple times without form changes in between). However, this impact would depend on whether the client can handle itext faster (I think it can, especially if the native XPath processor can be used).
- because of the first point it would become easier and more elegant to add support for external instances (itemsets).

Not a priority.

Client performance may be an issue, though reducing the # of DOM elements would have a somewhat positive impact.
